Automattic/wp-calypso
# Photos Picker: Introduce Google Photos Picker UI changes

Related to https://github.com/Automattic/wp-calypso/issues/95660

## Proposed Changes

* Update Google Photos Picker endpoint paths
* Introduce the Google Photos Picker Button component
* Integrate the Button into flow if the feature flag is turned on
* Implement Google Photos proxying while showing thumbnails

NOTE: I turned off the feature flag for DEV env as well since we don't want to screw the current Photos integration, so the other dev can work regular tasks

## Why are these changes being made?


* Adapting flow to support Google Photos Picker API integration

## Testing Instructions

* Turn on `google-photos-picker` feature flag
* Setup your sandbox and apply DIFF: D164673-code
* Go to `/media/SITE_SLUG`
* From the source dropdown, select Google Photos
* Click on "Open Google Photos Picker" button
* Make a selection in a different tab
* Check if the media library refreshes the screen and shows the list of selected files
* Select files to copy
* Click on "Copy to media library"
* Check if it's successfully copied
* Select "Google Photos" source again
* Check if the selection list is cleared

<img width="1728" alt="Screenshot 2024-11-01 at 17 23 42" src="https://github.com/user-attachments/assets/abc94a4c-0deb-4903-a00f-049290c0718a">
<img width="1728" alt="Screenshot 2024-11-01 at 17 24 01" src="https://github.com/user-attachments/assets/eceaa41b-7214-4d16-a7ef-99aa83b53fe7">
<img width="1728" alt="Screenshot 2024-11-01 at 17 24 29" src="https://github.com/user-attachments/assets/f6f13a11-65b3-4ebc-9fba-e2b5fd40fc21">
<img width="1728" alt="Screenshot 2024-11-01 at 17 24 44" src="https://github.com/user-attachments/assets/28d1bbd4-97c0-4a07-8f63-fe0434b02397">


## Pre-merge Checklist
